src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -493,6 +493,15 @@ def _normalize(self, text):
         normalizer = EnglishTextNormalizer(self.english_spelling_normalizer)
         return normalizer(text)
 
+    @staticmethod
```

Comment:
Useful to have this as a method of the tokeniser so that we can access it if we require it standalone (consistent with `_normalize` above)

Reply:
Mmm do you think we should have this added instead of using the `_normalize` function with by default the basic normalizer, or use an arg? Not sure we need both IMO. I don't see a case where someone would do `normalize` and `basic_normalize`